compiler/resolution/functionResolution.cpp

```diff
@@ -1770,8 +1962,13 @@ handleSymExprInExpandVarArgs(FnSymbol*  workingFn,
         // Otherwise, do the substitution now.
         subSymbol(workingFn->body, formal, var);
       }
+     } else {
+      // !needTupleInBody
+      substituteVarargTupleRefs(workingFn->body, n, formal, varargFormals);
+     }
 
```

Comment:
I avoid messing revision history on existing code whenever I reasonably can. Given that the only issue here is indenting by 1 space instead of 2 spaces, do you insist on doing so?